tests/dualtor_io/test_normal_op.py

```diff
@@ -20,137 +20,236 @@
 def test_normal_op_upstream(upper_tor_host, lower_tor_host,
                             send_server_to_t1_with_action,
                             toggle_all_simulator_ports_to_upper_tor,
-                            toggle_all_ports_both_tors_admin_forwarding_state_to_active,
+                            toggle_active_all_ports_both_tors,
                             cable_type):
     """Send upstream traffic and confirm no disruption or switchover occurs"""
-    send_server_to_t1_with_action(upper_tor_host, verify=True, stop_after=60)
     if cable_type == CableType.active_standby:
+        send_server_to_t1_with_action(upper_tor_host, verify=True, stop_after=60)
```

Comment:
We need to verify the upstream packets could be forwarded by both ToRs, but this will only verify traffic is forwarded by the upper ToR, right?

Reply:
Discussed offline. Resolving now.